.github/workflows/ci.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         id: set-matrix
         run: |
           # The output from rev-list ends with a newline, so we have to filter out index -1 in jq since it's an empty string
-          JQ_PIPELINE='split("\n") | .[0:-1] | map({base_ref: "${{ github.event.pull_request.base.ref }}", commit: .}) | select(length > 0) | {include: .}'
+          JQ_PIPELINE='split("\n") | .[0:-1] | map({commit: ., base_ref: "${{ github.event.pull_request.base.ref }}"}) | select(length > 0) | {include: .}'
```

Comment:
As we discussed offline, you could remove `base_ref: "${{ github.event.pull_request.base.ref }}"` and reference it directly instead of `matrix.base_ref` later.

Reply:
that would make my PR obsolete.
@nineinchnick do you want to send a PR with such change?
(be sure to include some empty commit, to exercise this within the PR)

Reply:
https://github.com/trinodb/trino/pull/15939

Reply:
thank you!